Update identifying-appropriate-positions.md (#239)
usds/SME-QA-Website: pages/phases/getting-started/identifying-appropriate-positions.md

### Identify Billets

> We recommend having at least three and ideally five or more opening/billets to fill in your hiring action. This ensures that the time SMEs spend effectively assessing applicants is leveraged through multiple hires. It also saves time for applicants and HR: if you're able to keep a certiciate open for a year and make 5+ hires off of one hiring action there is less of a need to have applicants apply to many different one-off vacancies.  Ideas if you do not have at least 3-5 openings:

Reviewer: ```suggestion
We recommend having at least three and ideally five or more opening/billets to fill in your hiring action. This ensures that the time SMEs spend assessing applicants is leveraged through multiple hires. It also saves time for applicants and HR: if you're able to keep a certificate open for a year and make 5+ hires off of one hiring action, you will need fewer one-off vacancy announcements.  Ideas if you do not have at least 3-5 openings:
```

> - Try a shared hiring agency with another office or agency with openings for the same position. This way there are more SMEs involved therefore decreasing the overall time burden on one agency's SMEs. USDS recently completed a SME-QA hiring action with 20 SMEs from 9 agencies hiring Customer Experience Specialists. OPM was hired as the HR Specialist and agencies were able to share the certificate the same day. If two agencies want to partner on their own, they are allowed to share the certificate as long as they have applicants opt-in when they apply and have a merit promotion announcement open for 10 days before the second agency makes selections. 

Reviewer: ```suggestion
- Try working with other offices or agencies with openings for the same position. This can also involve more SMEs and decrease the overall time burden on any one group of SMEs. USDS recently completed a SME-QA hiring action with 20 SMEs from 9 agencies hiring Customer Experience Specialists. OPM was hired as the HR Specialist and agencies were able to share the certificate the same day. If two agencies want to partner on their own, they are allowed to share the certificate as long as they have applicants opt-in when they apply and have a merit promotion announcement open for 10 days before the second agency makes selections. 
```

> - If you have 5+ openings for similiar roles that share the same core competetencise but have slightly different specialized experience required for each role, you can have the SMEs assess resumes and conduct the first interview to ensure the core competenies are met. The second interview can be dedicated to seeing which applicants meet the additional competency for each slightly different role. Separate certificates can be issued at the end of the hiring action for each sub-role within the announcement. The first two SME-QA pilots at HHS and DOI deployed this strategy since the participating offices all had the same core requirements but slightly different specialized experience for each sub-role.

Reviewer: ```suggestion
- If you have 5+ openings for similar roles that share the same core competencies but have slightly different specialized experience required for each role, you can have the SMEs assess resumes and conduct the first interview to ensure the core competencies are met. The second interview can be dedicated to seeing which applicants meet the additional competency for each slightly different role. Separate certificates can be issued at the end of the hiring action for each sub-role within the announcement. The first two SME-QA pilots at HHS and DOI deployed this strategy since the participating offices all had the same core requirements but slightly different specialized experience for each sub-role.
```

Reviewer: Do you want to mention parentheticals elsewhere and then link to that mention? It seems like "how to do parentheticals" is different than identifying positions.

> - If you are not going to make more than 2-3 offers from a SME-QA hiring action, you can iterate on the process to make it less time intensive: One option is to pull a ranking list between resume review and the first assessment round [using an exceeds score in resume review](/hiring-phases/reviewing-resumes/prep/#using-an-exceeds-score-in-resume-review). This way applicants are only conducting the first assessment with applicants who have superior resumes plus any veterans applicants. Normally, the SME-QA process has any applicants who meet the minimum qualification bar in resume review proceed to the next stage. Another option is adding a written prompt assessment, which requires less SME time to conduct and review. In SME-QA pilots with CMS and EPA, they used a written prompt assessment between the resume review and interview phase which resulted in only 15 interviews needed total at each agency rather than the average of 125 interviews needed in the first two pilots that did not have a written prompt assessment.

Reviewer: I feel like this content should go in the other sections, and be linked from here. (this paragraph is about assessment options). That ok?
